src/components/reportRepair/notEligibleNonEmergency.js

```diff
@@ -0,0 +1,26 @@
+import {
+  H1,
+  GridRow,
+  GridCol,
+  Paragraph
+} from 'govuk-react';
+
+const NotEligibleNonEmergency = () => {
```

Comment:
With the changes below, this component is now specific to communal repairs - please updated the component name and route/url.